source/installation/condafaq.rst

```diff
@@ -0,0 +1,92 @@
+:orphan:
+
+Conda Troubleshooting
+=====================
+
+Before using the troubleshooting below please ensure you are using Mambaforge. Using mamba is preferable over conda as it's faster and better at resolving dependencies.
+To install mambaforge please follow `this documentation <https://mamba.readthedocs.io/en/latest/installation/mamba-installation.html>`__
+
+Issues
+------
+
+- :ref:`Unable to find mantid or mantidworkbench <unable_to_find>`
+- :ref:`Installing on Apple silicon devices <osx_install>`
+- :ref:`Unable to resolve dependencies <dependency_fail>`
+- :ref:`404 error when trying to install <404_error>`
+
+.. _unable_to_find:
+
+Unable to find `mantid` or `mantidworkbench`
+############################################
+
+.. code-block:: sh
+	PackagesNotFoundError: The following packages are not available from current channels
+
+If you get this error it may be because you have not included the mantid channel in your download command. Please ensure you include `-c mantid` when installing.
+
+Example
+
+.. code-block:: sh
+
+   mamba create -n mantid_env -c mantid mantid
+
+You can also get this error when trying to install on Apple silicon devices. See :ref:`Installing on Apple silicon devices <osx_install>` for help.
```

Comment:
```suggestion
You may also encounter this error when trying to install on Apple silicon devices. See :ref:`Installing on Apple silicon devices <osx_install>` for help.
```

Not sure if this sentence is required because it's right next to the bit that it links to, but it's not a strong opinion.

Reply:
I guess I'm trying to future proof in case anything gets added inbetween!